internal/common/pointers.go

```diff
@@ -3,3 +3,10 @@ package common
 func ToPtr[T any](x T) *T {
 	return &x
 }
+
+func FromPtr[T any](ref *T) (value T) {
```

Comment:
Fwiw, we have a very similar function in https://github.com/osbuild/blueprint-schema/blob/main/pkg/ptr/any.go#L22 but its called `ValueOrEmpty` and has a docstring. I kinda like it slightly better as from FromPtr() its not clear what happens with nil pointers.

Reply:
oohhhh, i missed that one.